great_expectations/__init__.py

```diff
@@ -4,6 +4,24 @@
 __version__ = get_versions()["version"]  # isort:skip
 del get_versions  # isort:skip
 
+
+# 20210401 Will - Temporary fix to warn users against current incompatibility with Sqlalchemy 1.4.0
+# To be removed once a better fix is implemented
+
+import sys
+from great_expectations.exceptions.exceptions import GreatExpectationsError
+try:
+    import sqlalchemy as sa
+    sa_version = tuple(map(int, (sa.__version__.split("."))))
+    if sa_version >= (1, 4, 0):
+        raise GreatExpectationsError(f"""
```

Comment:
Do we use this pattern elsewhere? I don't know all the nuances of version numbers/potential edge cases, so would think using the `packaging` module would be safer. e.g.:

```
from packaging import version

if version.parse(sa.__version__) >= version.parse("1.4.0"):
    ...
```

Reply:
Thanks for this.. I was hesitant to introduce another package, but realized that it was already used elsewhere in GE.